packages/loot-core/src/shared/util.js

```diff
@@ -377,15 +377,23 @@ export function looselyParseAmount(amount) {
     return isNaN(v) ? null : v;
   }
 
+  function extractNumbers(v) {
+    return v.replace(/[^0-9-]/g, '');
+  }
+
+  if (amount.startsWith('(') && amount.endsWith(')')) {
+    amount = amount.replace('(', '-').replace(')', '');
+  }
+
   let m = amount.match(/[.,][^.,]*$/);
   if (!m || m.index === 0) {
-    return safeNumber(parseFloat(amount));
+    return safeNumber(parseFloat(extractNumbers(amount)));
   }
 
-  let left = amount.slice(0, m.index);
-  let right = amount.slice(m.index + 1);
+  let left = extractNumbers(amount.slice(0, m.index));
+  let right = extractNumbers(amount.slice(m.index + 1));
 
-  return safeNumber(parseFloat(left.replace(/[^0-9-]/g, '') + '.' + right));
+  return safeNumber(parseFloat(left + '.' + right));
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏I don't quite understand the need for this change. Why is this necessary? Previously only `left` was sanitized. Now both `left` and `right` is sanitized.